sklearn/naive_bayes.py

```diff
@@ -51,6 +51,14 @@ def _joint_log_likelihood(self, X):
         predict_proba and predict_log_proba.
         """
 
+    def _check_X(self, X):
+        """To be overridden in subclasses with the actual checks."""
+        # Note that this is not marked @abstractmethod as long as the
+        # deprecated public alias sklearn.naive_bayes.BayesNB exists
+        # (until 0.24) to preserve bacward compat for 3rd party projects
```

Comment:
typo, backward

Reply:
Thanks, I will push a fix with `[ci skip]` once the current CI run has ended.